### PR TITLE
Introduce an Inhibitor model

### PR DESCRIPTION
**Pull request by bocekm**
This model is to be used by an actor in case the actor decides that the upgrade process must be stopped.

@vinzenz's idea.

Update to How To doc is necessary upon merging.

### CODE REVIEW

**Comment by vinzenz** [comment]
@bocekm verifycheckresults needs to be updated to consume, CheckResult and Inhibitor

File: docs/howto-inhibit-upgrade.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+With latest changes on Leapp and with new actors added to the el7toel8 Leapp repository, any actor can inhibit the upgrade process by producing a specific message when a problem is found. The message model to use in this case is [Inhibitor](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/models/reports.py). The Inhibitor messages are collected in the Report phase, so if there's at least one Inhibitor message produced before this phase, the upgrade will be stopped in the Reports phase. It means that any Inhibitor message produced after the Report phase will have inhibiting effect. The details mentioned in the Inhibitor messages will be part of the report available to the user to review.
```
Comment (jmikovic) [inline docs/howto-inhibit-upgrade.md]: Inhibitor messages after Report phase will NOT have inhibiting effect, right?

Comment (vinzenz) [reply docs/howto-inhibit-upgrade.md]: @jmikovic  Correct

Comment (bocekm) [reply docs/howto-inhibit-upgrade.md]: Right, that's what I wanted to write there :) Thanks.

File: repos/system_upgrade/el7toel8/actors/verifycheckresults/actor.py [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
     name = 'verify_check_results'
     description = 'Verify results, stop process if error and generate report.'
-    consumes = (CheckResult,)
+    consumes = (CheckResult, Inhibitor)
     produces = ()
     tags = (ReportPhaseTag, IPUWorkflowTag)
 
     def process(self):
-        results = [msg for msg in self.consume(CheckResult)]
+        results = [msg for msg in self.consume(CheckResult)] + [msg for msg in self.consume(Inhibitor)]
```
[the remark this answers is not in the document]
Comment (bocekm) [reply repos/system_upgrade/el7toel8/actors/verifycheckresults/actor.py]: Done.

**Pull request merged**